module/main.go

```diff
@@ -21,7 +21,7 @@ var (
 )
 
 func main() {
-	utils.ContextualMain(mainWithArgs, golog.NewLogger("rplidarModule"))
+	utils.ContextualMain(mainWithArgs, module.NewLoggerFromArgs("rplidarModule"))
```

Comment:
@benjirewis just confirming: is this a reasonable way for go modules to initialize their logger so they can ensure they boot with the correct log level?